### ISSUE TITLE
段落缩排

### ISSUE THREAD

**Issue by xfq** [w3c/clreq#521]
https://www.w3.org/TR/clreq/#line_head_indent_of_paragraphs

> Generally speaking, the characters in the paragraphs which apply paragraph indent should be the same as the characters in the body content. Sometimes, due to the different typefaces, the size of characters in the paragraphs which apply paragraph indent differ from the characters in the body content. In this case, a certain amount of space might be added before and after the indent paragraph so as to make a clear distinction from other paragraphs. The space added is usually an integer times the height of paragraphs in the body content

> 一般而言以缩排处理的段落，文字尺寸与内文相同。有时会调整字体，与内文做出差异。或在前后段落间加入段落间距，使该段落与内文段落差异更为明显。加入间距所占空间应为内文行的整数倍。

英文版至少有两个地方需要改进：

1. 第一句话没有译出「文字尺寸」
2. 后面的the indent paragraph应为the indented paragraph

**Comment by xfq** [comment]
英文版修改如下（还修改了其他几处）：

> Generally speaking, the characters in the indented paragraphs should have the same font size as the characters in the body content. Sometimes, due to differences in typefaces, the size of characters in the paragraphs which apply paragraph indent differs from the characters in the body content. In this case, a certain amount of space might be added before and after the indented paragraph to distinguish them clearly from other paragraphs. The space added is usually an integer multiple of the height of paragraphs in the body content.

> 一般而言以缩排处理的段落，文字尺寸与内文相同。有时会调整字体，与内文做出差异。或在前后段落间加入段落间距，使该段落与内文段落差异更为明显。加入间距所占空间应为内文行的整数倍。

**Comment by r12a** [comment]
Typo correction:

> Generally speaking, the characters in the indented paragraphs should have the same font size as the characters in the body content. Sometimes, due to differences in typefaces, the size of characters in the paragraphs which apply a paragraph indent differ from the characters in the body content. In this case, a certain amount of space might be added before and after the indented paragraph to distinguish it clearly from other paragraphs. The space added is usually an integer multiple of the height of the paragraphs in the body content.

**Comment by xfq** [comment]
"The size" is singular. If there's a typo, maybe we should make "the size" plural, instead of changing "differs" to "differ"?

**Comment by r12a** [comment]
You're right about the s.  That sentence is a bit hard to parse though. I wonder whether this might be better?

> Sometimes, however, paragraphs that apply a paragraph indent may have a different typeface and therefore character size from that of the body content.

Just an idea...

**Comment by xfq** [comment]
OK, so now the new version is:

> Generally speaking, the characters in the indented paragraphs should have the same font size as the characters in the body content. Sometimes, however, paragraphs that apply a paragraph indent may have a different typeface and therefore character size from that of the body content. In this case, a certain amount of space might be added before and after the indented paragraph to distinguish it clearly from other paragraphs. The space added is usually an integer multiple of the height of the paragraphs in the body content.
